### DISCUSSION TITLE
Cookies not allowed

### DISCUSSION THREAD

**Discussion by danielpirs** [langfuse/langfuse#1672] [Support]
Hi, I am using langfuse hosted on azure containers. 
Unfortunately browser cookies for my domain ('azurecontainerapps.io') are not allowed by corporate policy and therfore following error appers in browser
![image](https://github.com/langfuse/langfuse/assets/72087218/480f9d32-cfa3-4b3a-8a86-478ad48c7a9c)
Does anyone has some suggestion for a workaround or something?
Thanks!

**Comment by marcklingen** [comment]
Langfuse needs cookies for frontend authentication (storing a JWT). Any chance you can circumvent this restriction -- e.g. by deploying on a domain owned by your org?

**Comment by danielpirs** [comment]
Hi Marc,
first let me use the opportunity to thank you for you work on langfuse, it is a great tool. We are just starting with it, but it is very promising.
In a large corp is not easy to get an approval to deploy it on our own domain. But it looks like there is no other option.
Thanks,
Daniel

**Reply by marcklingen** [reply]
Thank you 🙏 

Hope you figure out a way to make this login cookie work. If you've got a lot of services running self-hosted, I guess this isn't the first time this issue's popped up and there might be some standard procedure for it.

**Comment by danielpirs** [comment]
Ok, investigated more and found out that is not about cookies. But there was also some misunderstanding:
1. It is not about langfuse web site itslef, the error is appearing when I create Langfuse object with 'new Langfuse(...)' in my code 
2. I am using JS/TS SDK, not the Python one. My code (app) runs in browser
3. Everything works fine with Edge and Chrome, and Firefox (124.0.2 (64-bit)) on my private laptop. Also Edge and Chrome on my work laptop and behind firewall work fine. Only Firefox on my work laptop is causing issues (115.9.1esr (64-bit)). It is probably with some Firefox confing but I am not able to find which one (or maybe with firefox version). The code where error appears (screenshot in my first post) points to something with local storage, but I am not able to find which one.

I know it is a tough problem to crack and there is a good workaround (another browser). But just leaving it here if anyone has similar problem.

**Reply by marcklingen** [reply]
I think I don’t fully understand, how does the browser relate to the Langfuse JS SDK which is run server-side. Do you mean LangfuseWeb or something else?

**Reply by danielpirs** [reply]
I am not runing Langfuse JS SDK server side, currently I run my whole application in React frontend. 
I use Azure JS SDK to communicate with Azure OpenAI hosted models (and Azure AI Search Service) and Langufse JS SDK for monitoring - everything in React frontend.
This is just a first POC/Pilot solution, when/if proven good in real world usage (and that is why we use Langufuse, for monitoring and evaluation), we are planning to move this to the backend (probably Azure Functions and use Python SDK or Langchain instead)
